### ISSUE TITLE
v1 website bugs

### ISSUE THREAD

**Issue by carolynvs** [getporter/porter#1767]
- [ ] The base url passed to hugo is https://release-v1--porter.netlify.app instead of https://release-v1.porter.sh, so it shows up with the wrong URL in our meta tags.
- [ ] FontAwesome doesn't load

**Comment by joshuabezaleel** [comment]
I've actually also found several bugs and typos on the websites when browsing it. Is it okay if I list them on this issue or should I create a separate new one? @carolynvs

**Comment by carolynvs** [comment]
Yes please list them on this issue. We can split it out into separate issues later if needed.

**Comment by joshuabezaleel** [comment]
- [ ] Typo of "Hashicorp"
![image](https://user-images.githubusercontent.com/7043511/134451144-65176d9b-d2ff-4723-918f-a8616005967d.png)

- [ ] The page was about credential set (porter.sh/quickstart/credentials) but the following sentence mentioned parameter set and uses `porter parameters show` command instead
![image](https://user-images.githubusercontent.com/7043511/134451244-65591884-0575-4b02-a4a0-9ad1fdce4b87.png)

- [ ] Installation name should be `porter-hello` instead of just `hello`
![image](https://user-images.githubusercontent.com/7043511/134451308-91fb2adf-865d-4cf9-b5c9-a53b70657a99.png)

**Comment by carolynvs** [comment]
Thanks @joshuabezaleel!

**Comment by carolynvs** [comment]
Closing since I think all issues have been addressed. If not, let's open separate issues for anything we find at this point.
